autograder/submitty_autograding_shipper.py

```diff
@@ -37,6 +37,58 @@
 JOB_ID = '~SHIP~'
 
 
+# This function is copied, would rather use the version in this file instead.
+# import sys
+# sys.path.append('/usr/local/submitty/sbin/shipper_utils/')
+# import ssh_jump_proxy
+def ssh_connection_allowing_jump_proxy(target_user, target_host):
+
+    # load and parse the ssh config
+    my_key_filename = None
+    intermediate_host = None
+    intermediate_user = None
+    config_file = os.path.expanduser("~")+"/.ssh/config"
+    if (os.path.exists(config_file)):
+        conf = paramiko.SSHConfig()
+        conf.parse(open(config_file))
+        o = conf.lookup(target_host)
+        if 'proxyjump' in o:
+            (intermediate_user, intermediate_host) = str(o['proxyjump']).split('@')
+        if 'identityfile' in o:
+            my_key_filename = o['identityfile'][0]
```

Comment:
Is it possible for `o['proxyjump']` or `o['identityfile']` to be formatted in an unexpected way? (specifically, missing an `@` or not having index `0`? If so, does this need to go inside of a try/except, or do we expect that calls to this function should always be wrapped in a try/except?

Reply:
Yes, I should add error checking around these items...